### ISSUE TITLE
Relax package dependencies where possible

### ISSUE THREAD

**Issue by tvasenin** [LexPredict/lexpredict-lexnlp#69]
Is it possible to relax requirements (at least for the more stable packages) in order to get less dependency conflicts when installing?

**Comment by huister1216** [comment]
yes please someone relax these requirements a bit to allow for minor updates from other packages.
 
People will most likely be using this in conjunction with other libraries on pip to build something and lexnlp keeps creating errors on pip. 

The python-requirements.txt, python-requirements-dev.txt, and python-requirements-full files all have exact versions of each dependency package which creates problems most likely for many people.

Examples:
ERROR: lexnlp 2.3.0 has requirement beautifulsoup4==4.11.1, but you'll have beautifulsoup4 4.11.2 which is incompatible.
ERROR: lexnlp 2.3.0 has requirement scikit-learn==0.24, but you'll have scikit-learn 0.24.2 which is incompatible.
ERROR: lexnlp 2.3.0 has requirement tqdm==4.64.1, but you'll have tqdm 4.65.0 which is incompatible.
